doc/source/whatsnew/v1.5.0.rst

```diff
@@ -279,6 +279,7 @@ Other enhancements
 - :class:`Series` reducers (e.g. ``min``, ``max``, ``sum``, ``mean``) will now successfully operate when the dtype is numeric and ``numeric_only=True`` is provided; previously this would raise a ``NotImplementedError`` (:issue:`47500`)
 - :meth:`RangeIndex.union` now can return a :class:`RangeIndex` instead of a :class:`Int64Index` if the resulting values are equally spaced (:issue:`47557`, :issue:`43885`)
 - :meth:`DataFrame.compare` now accepts an argument ``result_names`` to allow the user to specify the result's names of both left and right DataFrame which are being compared. This is by default ``'self'`` and ``'other'`` (:issue:`44354`)
+- :class:`Interval` now supports checking whether one interval is inside of another interval (:issue:`46613`)
```

Comment:
```suggestion
- :class:`Interval` now supports checking whether one interval is contained by another interval (:issue:`46613`)
```

Reply:
Hm, I wonder why this addition seems to make the documentation fail to build.